sql/burnham_derived/test_result_v1/init.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE
+  burnham_derived.test_result_v1(
+    submission_timestamp TIMESTAMP,
+    test_run STRING,
+    test_name STRING,
+    test_outcome STRING,
+    test_duration_seconds FLOAT64,
```

Comment:
Floats can be a bit of a pain to work with, particularly when converting between JSON and other formats. Can we consider changing this to an INT64 called `test_duration_millis` or `test_duration_micros`?

Reply:
Yes, we can change this to an `INT64` called `test_duration_millis` or `test_duration_milliseconds`.

The most recent test run took `4 passed in 3.82s`. That granularity is good enough!